data-binning/source/Program.cs

```diff
@@ -1,31 +1,104 @@
 ﻿// See https://aka.ms/new-console-template for more information
 using System.ComponentModel.DataAnnotations;
 using System.Net.Http;
+using Microsoft.Azure.Cosmos;
 using Newtonsoft.Json;
+using static Azure.Core.HttpHeader;
+using Microsoft.Extensions.Configuration;
+using Container = Microsoft.Azure.Cosmos.Container;
+using Database = Microsoft.Azure.Cosmos.Database;
 
-namespace Bucketing
+namespace data_binning
 {
     internal class Program 
     {
-        static string urlBase = "http://localhost:7071"; // "http://<functionapp>.azurewebsites.net:7071"
- 
-        public static async Task<string> SimulateEvents(int deviceCount, int timeout)
+
+
+
+        static DateTime TruncateToMinute(DateTime time)
+        {
+            return new DateTime(time.Year, time.Month, time.Day, time.Hour, time.Minute, 0, DateTimeKind.Utc);
+        }
+
+        public static async Task SimulateEvents(int deviceCount, int timeout)
         {
-            HttpClient client = new HttpClient();
-            client.Timeout = TimeSpan.FromMinutes(10);
+            Database db;
+            Container container;
+            string partitionKeyPath = "/DeviceId";
+
+            var configuration = new ConfigurationBuilder()
+                 .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
+                 .AddJsonFile($"appsettings.{Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT")}.json", optional: true);
+
+            var config = configuration.Build();
+
+
+            string uri = config["CosmosUri"];
+            string key = config["CosmosKey"];
+
+            CosmosClient client = new(
+                accountEndpoint: uri!,
+                authKeyOrResourceToken: key!);
+
 
-            var url = $"{urlBase}/api/CosmosPatternsBucketingExample";
-            var args = new Dictionary<string, int>
+            string databaseName = "Hotels";
```

Comment:
Change database to "CosmosPatterns". change container to "DataBinning". I'm trying to consolidate everything in a single database with shared throughput so that users can run multiples of these as cheaply as possible.